docs/docs/constraints.md

```diff
@@ -54,7 +55,7 @@ $ curl -X POST -H "Content-type: application/json" localhost:8080/v2/apps -d '{
   }'
 ```
 
-Alternatively, you could specify that all of the app tasks should run on the same rack, but without specifying which rack:
+This example leaves the attribute field empty. This tells Marathon that all of the app tasks should run on the same rack, but does not specify which.
```

Comment:
"This example leaves the value field empty."

Because the attribute is "rack_id"